wireless_modules/middle_module/main.py

```diff
@@ -0,0 +1,91 @@
+# Note: You can remove all the PRINT statements when everything is already tested
```

Comment:
Probably don't need this warning?

Reply:
I usually keep notes at the top of the code reminding what was only part of testing or temporary development use and can bee removed later. this way I can just control-F and delete them at the end. Not exactly a warning but in case we don't want print statements clouding the output later on.

Reply:
I actually use this to highlight keywords such as `NOTE:`

https://marketplace.visualstudio.com/items?itemName=wayou.vscode-todo-highlight

Reply:
Interesting, I'll have a look into it thanks!